Makefile

```diff
@@ -0,0 +1,37 @@
+BRANCH      ?= $(shell git rev-parse --abbrev-ref HEAD)
```

Comment:
I'm questioning the need for this file? It looks a little over-engineered. If we go with Makefiles, please make sure we have a similar one for all our services. Otherwise I would not introduce this here.

Reply:
I would highly recommend to add the Makefile also for all our other services. I think it's always good to pass the version, etc. into the binary and log the version during the start of a service.